amqphandler/amqphandler.go

```diff
@@ -60,6 +60,8 @@ const (
 type AmqpHandler struct { //nolint:stylecheck
 	sync.Mutex
 
+	disableLogsTrim bool
```

Comment:
Don't see why we need to have it configurable. Let's just remove log trimming at all.

Reply:
JIRA task: https://jiraeu.epam.com/browse/EPMPAOS-5351
Title: [AOS Core] Add the ability not to truncate the unit's messages to the cloud.

So the task is to add an ability to not truncate logs

Reply:
Agreed to not truncate logs at all.
Updated PR.